CONTRIBUTING.md

```diff
@@ -7,30 +7,34 @@ We'd love to get patches from you!
 The workflow that we support:
 
 1.  Fork finagle
-2.  Make a feature branch
-3.  Make your cool new feature or bugfix on your branch
-4.  Write a test for your change
-5.  From your branch, make a pull request against twitter/finagle/master
-6.  Work with repo maintainers to get merged in
-7.  Wait for your change to be pulled into twitter/finagle/master
-8.  Merge twitter/finagle/master into your origin master
-9.  Delete your feature branch
+2.  Check out the `develop` branch
+3.  Make a feature branch
+4.  Make your cool new feature or bugfix on your branch
```

Comment:
I guess we probably should, although once the new round of releases comes out it shouldn't be necessary for a while, and once we've started publishing snapshots it'll be unnecessary for good. I could go either way.